### PR TITLE
support repeated native plugins with different options

### PR DESCRIPTION
**Pull request by bjaglin**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -15,7 +15,7 @@ scalaVersion := "2.12.13"
 addSbtPlugin("org.portable-scala" % "sbt-platform-deps" % "1.0.0")
 
 libraryDependencies ++= Seq(
-  "com.thesamet.scalapb" %% "protoc-bridge" % "0.9.1"
+  "com.thesamet.scalapb" %% "protoc-bridge" % "0.9.1+1-c8799d87-SNAPSHOT"
```
Comment (bjaglin) [inline build.sbt:18]: https://github.com/scalapb/protoc-bridge/pull/158

File: build.sbt [hunk 2]
```diff
@@ -15,7 +15,7 @@ scalaVersion := "2.12.13"
 addSbtPlugin("org.portable-scala" % "sbt-platform-deps" % "1.0.0")
 
 libraryDependencies ++= Seq(
-  "com.thesamet.scalapb" %% "protoc-bridge" % "0.9.1"
+  "com.thesamet.scalapb" %% "protoc-bridge" % "0.9.1+4-8a125376-SNAPSHOT"
```
Comment (bjaglin) [inline build.sbt:18]: Pending release of https://github.com/scalapb/protoc-bridge/pull/160

File: src/main/scala/sbtprotoc/ProtocPlugin.scala [hunk 3]
```diff
@@ -516,7 +516,13 @@ object ProtocPlugin extends AutoPlugin {
                 f.getAbsolutePath()
               }
           }
-        s"--plugin=${dep.name}=${pluginPath}"
+        }
+        val targetCount = PB.targets.value.count {
+          case Target(PluginGenerator(name, _, _), _, _) => s"protoc-gen-$name" == dep.name
+          case _                                         => false
+        }
+        if (targetCount == 1) Seq(s"--plugin=${dep.name}=${pluginPath}")
```
Comment (bjaglin) [inline src/main/scala/sbtprotoc/ProtocPlugin.scala:524]: see discussion in https://github.com/scalapb/protoc-bridge/pull/160#discussion_r566327478

**Comment by thesamet** [comment]
rebased

**Comment by bjaglin** [comment]
fixed the scripted on windows

**Pull request merged**